ProcessLib/HeatTransportBHE/HeatTransportBHEProcess.cpp

```diff
@@ -239,16 +295,44 @@ void HeatTransportBHEProcess::createBHEBoundaryConditionTopBottom(
             for (auto const& in_out_component_id :
                  bhe.inflow_outflow_bc_component_ids)
             {
-                // Top, inflow.
-                bcs.addBoundaryCondition(
-                    createBHEInflowDirichletBoundaryCondition(
-                        get_global_bhe_bc_indices(bc_top_node_id,
-                                                  in_out_component_id),
+                if (bhe.usePythonBC)
+                // call BHEPythonBoundarycondition
```

Comment:
drop

Reply:
Actually this is necessary. I come over and explain it to you.

Reply:
Sorry, I was not precise.
I think the comment `// call BHEPyBC` does not add to understanding of the code...
Especially there is no "call" to python BC, but creation.

Reply:
?